config/config.go

```diff
@@ -333,6 +334,10 @@ func ValidateStoreAndDistroChanges(currentStoreType, previousStoreType StoreType
 }
 
 func (c *Config) IsProFeatureEnabled() bool {
+	if os.Getenv("SKIP_VALIDATE_PRO_FEATURES") == "true" {
```

Comment:
where is this one set and why is it needed?

Reply:
its mostly for development to skip the pro checks when the cli is not authenticated / logged in